### PR TITLE
Feature/error suspense card

### PR DESCRIPTION
**Pull request by HurricanKai**

### CODE REVIEW

File: src/components/LoadingCard/LoadingCard.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.loading {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    z-index:0;
+    border-radius: 15px; /* Sync with Card */
```
Comment (7rebux) [inline src/components/LoadingCard/LoadingCard.css:7]: Maybe make an top level css file with that value

File: src/components/LoadingCard/LoadingCard.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        left:0;
+        right:-100%;
+        bottom:0;
+        animation: aaa 5s linear infinite;
+        background-size: 200%;
+        background: linear-gradient(0.25turn, #eee, #fff, #eee) left/50% 100%;
+    }
+}
+
+@keyframes aaa {
```
Comment (7rebux) [inline src/components/LoadingCard/LoadingCard.css:29]: aaa

Comment (HurricanKai) [reply src/components/LoadingCard/LoadingCard.css:29]: What should I name this? It's just a local animation and "loading" is taken

File: src/components/LoadingCard/LoadingCard.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import { Meta, Story } from '@storybook/react';
+
+import LoadingCard from './LoadingCard';
+
+export default {
+  title: 'Library/Loading Card',
+  component: LoadingCard,
+} as Meta;
+
+const Template: Story = (args) => <LoadingCard {...args} />;
```
Comment (7rebux) [inline src/components/LoadingCard/LoadingCard.stories.tsx:12]: Template is not needed here if there is only one variant

File: src/components/LoadingCard/LoadingCard.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Meta, Story } from '@storybook/react';
+
+import LoadingCard from './LoadingCard';
+
+export default {
+  title: 'Library/Loading Card',
```
Comment (7rebux) [inline src/components/LoadingCard/LoadingCard.stories.tsx:8]: Maybe put all cards in a cards folder

File: src/components/RootLayoutCard/RootLayoutCard.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import RootLayoutCard, { RootLayoutCardProps } from './RootLayoutCard';
+
+export default {
+  title: 'Library/Root Layout Card',
+  component: RootLayoutCard,
+} as Meta;
+
+const Template: Story<RootLayoutCardProps> = (args) => <RootLayoutCard {...args} />;
+
+const Throw = () => { throw new Error("AAA"); };
```
Comment (7rebux) [inline src/components/RootLayoutCard/RootLayoutCard.stories.tsx:14]: AAA

File: src/components/RootLayoutCard/RootLayoutCard.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import React, { Suspense } from "react";
+import Card from "../Card";
+import { CardProps } from "../Card/Card.types";
+import ErrorBoundaryCard, { ErrorBoundaryCardProps } from "../ErrorBoundaryCard/ErrorBoundaryCard";
```
Comment (7rebux) [inline src/components/RootLayoutCard/RootLayoutCard.tsx:4]: Add an index file and put props inside a seperate file

File: src/components/TitelizedCard/TitelizedCard.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+.title-wrapper .title {
+    font-size: 26px;
+    font-weight: 600;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+      Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.content {
+    font-weight: 300;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```
Comment (7rebux) [inline src/components/TitelizedCard/TitelizedCard.css:17]: Font family can be defined inside `.title-wrapper`

**Review by 7rebux — CHANGES_REQUESTED** [review]
Format each file using the given prettier config

File: src/components/LoadingCard/LoadingCard.css (hunk 2 above)
Comment (7rebux) [reply src/components/LoadingCard/LoadingCard.css:29]: `animation` or something

**Review by 7rebux — APPROVED** [review]

**Pull request merged**